README.md

```diff
@@ -26,22 +26,18 @@ TODO: Add a link to download the JAR directly for e.g. adding to the Spark shell
 The spark-avro jar file can also be added to a Spark using the `--jars` command line option.  For example, to include it when starting the spark shell:
 
 ```
-$ bin/spark-shell --jars spark-avro_2.10-0.1.jar
+$ bin/spark-shell --jars spark-avro_2.10-1.0.0.jar
+```
 
-Spark assembly has been built with Hive, including Datanucleus jars on classpath
-Welcome to
-      ____              __
-     / __/__  ___ _____/ /__
-    _\ \/ _ \/ _ `/ __/  '_/
-   /___/ .__/\_,_/_/ /_/\_\   version 1.2.0
-      /_/
+There is a version of this library for spark 1.2, which has the following coordinates:
```

Comment:
Looking at the README, one could misread and accidentally pick the second. I think we should just say 
"For use with Spark 1.2, you can use version `0.2.0` instead."